src/Common/src/Interop/OSX/Interop.Libraries.cs

```diff
@@ -4,7 +4,7 @@
 
 internal static partial class Interop
 {
-    private static partial class Libraries
+    internal static partial class Libraries
```

Comment:
What is the reason this needed to change from private to internal? I don't see any access to the type outside of Interop in this PR.